certchain/ocsp_response.go

```diff
@@ -53,6 +53,9 @@ var DummyOCSPResponse *OCSPResponse = &OCSPResponse{
 func ParseOCSPResponse(bytes []byte) (*OCSPResponse, error) {
 	resp, err := ocsp.ParseResponse(bytes, nil)
 	if err != nil {
+		if string(bytes) == "dummy-ocsp" {
+			return DummyOCSPResponse, nil
```

Comment:
Add a comment to explain this is necessary to serve a(n invalid) cert-chain+cbor in AllowTestCert mode.

Reply:
Done.